When using tailwind-config option, accept paths based on root of importer

**Goal**

Say a have a project with the structure like this:

```
/my-project
  package.json
  tailwind.config.js
```

I would like to be able to call:

```
elm-tailwind-modules --tailwind-config ./tailwind.config.js
```

**Problem**

Currently, I need to specify to full relative path based on the internal structure of the publish elm-tailwind-modules package:

```
elm-tailwind-modules --tailwind-config ../../../tailwind.config.js
```

This is not as intuitive and may lead to problems – e.g. if you change your internal folder structure, this relative path would not work anymore.

**Nice to have**

It would be even better if elm-tailwind-modules automatically detected if I had a `tailwind.config.js` at the root of my project folder (as this is the default) and if I wanted to specify a different one, I would pass in this extra option (maintaining the same path behavior previously described).

I'm more than happy to help out with a PR here! 🙂 If this is desirable, just let me know!
Cheers and props for the great work!

I'm using `elm-tailwind-modules` through an npm script. When running it without all the relative path stuff it results in this error:

```bash
Error: Cannot find module './tailwind.config.js'
Require stack:
- /###/node_modules/elm-tailwind-modules/dist/cli.js
```

So I need to require it in relation to `node_modules/elm-tailwind-modules/dist` .
BTW I'm running on a macOS Big Sur (but I don't think that matters that much… is just usual node stuff)

I can take a look at it this weekend! 🙂 I will open a PR if I find anything.

--- a/src/cli.ts
+++ b/src/cli.ts
@@ -1,5 +1,6 @@
 import * as elmTailwindModules from "./index";
 import { program } from "commander";
+import { join } from "path";
 
 program.name("elm-tailwind-modules");
 program.version("0.3.0");
@@ -13,7 +14,9 @@ program.option("--with-docs", `Add documentation to the generated modules (defau
 
     const options = program.opts();
 
-    const tailwindConfig = options.tailwindConfig == null ? elmTailwindModules.defaultTailwindConfig : await import(options.tailwindConfig);
+    const tailwindConfig = options.tailwindConfig == null
+        ? elmTailwindModules.defaultTailwindConfig
+        : await import(join(process.cwd(), options.tailwindConfig));
 
     elmTailwindModules.run({
         directory: options.dir,

